bio/diamond/blast/wrapper.py

```diff
@@ -0,0 +1,64 @@
+__author__ = "N. Tessa Pierce"
+__copyright__ = "Copyright 2019, N. Tessa Pierce"
+__email__ = "ntpierce@gmail.com"
+__license__ = "MIT"
+
+from os import path
+from snakemake.shell import shell
+
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+pep_query = snakemake.input.get("pep_query")
+nucl_query = snakemake.input.get("nucl_query")
+# assert pep_query is not None or nucl_query is not None, "either pep_query or nucl_query are required as input"
+queryfiles = [pep_query, nucl_query]
+assert (
+    list(map(bool, queryfiles)).count(True) == 1
+), "one query is required: either a peptide query (pep_query) for diamond blastp OR a nucleotide query (nucl_query) for diamond blastx"
```

Comment:
Maybe this is a bit shorter (and hopefully works, I haven't tested):
```suggestion
assert list(map(bool, [pep_query, nucl_query])).count(True) == 1,
    "one query is required: either a peptide query (pep_query) for diamond blastp OR a nucleotide query (nucl_query) for diamond blastx"
```